modules/gltf/test/data/meshopt/README.md

```diff
@@ -1 +1,3 @@
 - pirate.{glb,obj} - MIT License, from https://github.com/zeux/meshoptimizer/tree/master/demo
+
+- Lantern_meshopt.glb - created from [Lantern.glb](https://github.com/KhronosGroup/glTF-Sample-Models/blob/master/2.0/Lantern/glTF-Binary/Lantern.glb) with [meshoptimizer](https://github.com/zeux/meshoptimizer)
```

Comment:
10 MB test file is fairly big. Does it have other extensions so we can use it for more tests?

Reply:
I've replaced it with a smaller model: https://github.com/KhronosGroup/glTF-Sample-Models/blob/master/2.0/BoxTextured/glTF-Binary/BoxTextured.glb